mantle/platform/platform.go

```diff
@@ -403,9 +404,20 @@ func checkSystemdUnitFailures(output string, distribution string) error {
 		ignoredUnits = append(ignoredUnits, "user-runtime-dir@1000.service")
 	}
 
+	switch distribution {
+	case "rhcos":
+		unitNameIndex = 0
+		break
```

Comment:
The break is redundant here - unlike C++ and other languages, the break is provided for free.